config/config.go

```diff
@@ -304,6 +304,8 @@ type Config struct {
 	MuteTimeIntervals []MuteTimeInterval `yaml:"mute_time_intervals,omitempty" json:"mute_time_intervals,omitempty"`
 	TimeIntervals     []TimeInterval     `yaml:"time_intervals,omitempty" json:"time_intervals,omitempty"`
 
+	GoogleCloud GoogleCloudConfig `yaml:"google_cloud,omitempty" json:"google_cloud,omitempty"`
```

Comment:
suggestion: add a corresponding unit test in config/config_test.go

To ensure unmarshaling and validation works of a YAML string literal of the new config surface.

Reply:
Done.